src/kit.ts

```diff
@@ -318,7 +324,7 @@ export async function vsInstallations(): Promise<VSInstallation[]> {
 
   const vs_installs = JSON.parse(vswhere_res.stdout) as VSInstallation[];
   for (const inst of vs_installs) {
-    if (inst_ids.indexOf(inst.instanceId) < 0) {
+    if ((inst_ids.indexOf(inst.instanceId) < 0) && (inst.displayName)) {
```

Comment:
This line makes the problem. Visual studio 2015 and older do not provide a displayName. So all kit other then VS2017 are dropped.  In a later step are the displayName generated if necessary.

I think this modification does not belong to the PR.

Reply:
Nice find. I remember a PR several months ago that fixed this exact `displayName` issue. It's good then that we didn't just pass it through.

@schultetwin1 would you look into this?

Reply:
@KoeMai  you are correct, taking that out now. Sorry about that.